frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -181,7 +182,7 @@ export const eventUsageLogic = kea<
             const { created_at, name, is_shared, pinned, creation_mode } = dashboard
             const properties: Record<string, any> = {
                 created_at,
-                name: userLogic.values.user?.is_multi_tenancy ? name : undefined, // Don't send name on self-hosted
+                name: preflightLogic.values.preflight?.cloud ? name : undefined, // Don't send name on self-hosted
```

Comment:
It feels a bit excessive to load the preflight logic every time a dashboard is accessed? Could we cheat here and just check `window.location.href` for `app.posthog.com`?

Reply:
The cheat fails locally though, which is not ideal. 🤔 It's not a super light solution, but I actually think we should use the preflight endpoint more, as `user` really shouldn't have all these instance config properties.

Reply:
The cheat fails locally though, which is not ideal. 🤔 It's not a super light solution, but I actually think we should use the preflight endpoint more, as `user` really shouldn't have all these instance config properties.

Reply:
How often do you have `multi_tenancy` enabled and `self_capture` disabled on localhost? Even then, I guess it'd be fine to not send the name.

Reply:
Agreed, we can get away without the name